Doesn't build

```
$ make
CC main.c
In file included from /usr/include/readline/history.h:35:0,
                 from main.c:3:
/usr/include/readline/rltypedefs.h:71:28: error: unknown type name ‘FILE’
 typedef int rl_getc_func_t PARAMS((FILE *));
                            ^
/usr/include/readline/readline.h:429:20: error: unknown type name ‘FILE’
 extern int rl_getc PARAMS((FILE *));
                    ^
In file included from main.c:4:0:
/usr/include/readline/readline.h:558:8: error: unknown type name ‘FILE’
 extern FILE *rl_instream;
        ^~~~
/usr/include/readline/readline.h:559:8: error: unknown type name ‘FILE’
 extern FILE *rl_outstream;
        ^~~~
/usr/include/readline/readline.h:588:8: error: unknown type name ‘rl_getc_func_t’
 extern rl_getc_func_t *rl_getc_function;
        ^~~~~~~~~~~~~~
/usr/include/readline/readline.h:917:3: error: unknown type name ‘FILE’
   FILE *inf;
   ^~~~
/usr/include/readline/readline.h:918:3: error: unknown type name ‘FILE’
   FILE *outf;
   ^~~~
make: *** [Makefile:27: out/obj/main.o] Error 1

```

reordering the includes make it work again...

--- a/main.c
+++ b/main.c
@@ -1,10 +1,10 @@
 #include <getopt.h>
 #include <libusb.h>
+#include <stdio.h>
 #include <readline/history.h>
 #include <readline/readline.h>
 #include <stdbool.h>
 #include <stdlib.h>
-#include <stdio.h>
 #include <string.h>
 
 #define XILDEBUG_VID 0x1209

--- a/main.c
+++ b/main.c
@@ -300,6 +300,9 @@ int main(int argc, char *argv[])
 	while(!quit) {
 		line = readline("# ");
 
+		if(line == NULL)
+			break;
+
 		char *input = strtrim(line);
 		if(input) {
 			add_history(input);

